doc/modules/grid_search.rst

```diff
@@ -79,6 +79,10 @@ evaluated and the best combination is retained.
       classifier (here a linear SVM trained with SGD with either elastic
       net or L2 penalty) using a :class:`pipeline.Pipeline` instance.
 
+    - See :ref:`example_model_selection_plot_nested_cross_validation_iris.py`
```

Comment:
Add "This is best practice for evaluating the performance of a model with grid search."